src/components/Collections/ListItems/PlioListItem.vue

```diff
@@ -135,36 +81,64 @@ export default {
       actionButtonClass:
         "bg-gray-100 hover:bg-gray-200 rounded-md shadow-md h-10 ring-primary",
       urlCopyButtonClass: "text-yellow-600",
-      showActionButtons: true, // whether to show the action buttons
-      // whether the visibility of the action buttons has been manually set
-      hasUserSetActionVisibility: false,
+      scrollY: window.scrollY // the number of pixels scrolled vertically
     };
   },
-
   async created() {
     // load the plio only if the plio id is not empty
     if (this.isPlioIdValid) await this.loadPlio();
 
-    // determine the screen orientation when the list item is created
-    this.checkScreenOrientation();
-    // add listener for screen size being changed
-    window.addEventListener("resize", this.checkScreenOrientation);
+    // add listener for scrolling
+    window.addEventListener('scroll', this.handleScroll);
   },
-
   unmounted() {
     // remove listeners
-    window.removeEventListener("resize", this.checkScreenOrientation);
+    window.removeEventListener('scroll', this.handleScroll);
   },
-
   computed: {
     ...mapState("auth", ["activeWorkspace"]),
     ...mapState("sync", ["pending"]),
     ...mapState("plioItems", ["allPlioDetails"]),
-    toggleIconClass() {
-      return [
-        { "transform rotate-180": this.showActionButtons },
-        "transition ease duration-800",
-      ];
+
+
+    plioActionOptions() {
+      // the list of action buttons
+      let options = [
+        {
+          value: "edit",
+          label: this.$t("home.table.plio_list_item.buttons.edit"),
+          icon: "edit.svg",
+        },
+        {
+          value: "play",
+          label: this.$t("home.table.plio_list_item.buttons.play"),
+          icon: "play.svg",
+          disabled: !this.isPublished
+        },
+        {
+          value: "share",
+          label: this.$t("home.table.plio_list_item.buttons.share"),
+          icon: "share.svg",
+          disabled: !this.isPublished
+        }
+      ]
+      if (this.isTouchDevice) {
+        options.push({
+          value: "analyse",
+          label: this.$t("home.table.plio_list_item.buttons.analyse"),
+          icon: "analyze.svg",
+          disabled: !this.isPublished
+        })
+      }
+      let moreOptions = [
+        {
+          value: "duplicate",
+          label: this.$t("home.table.plio_list_item.buttons.duplicate"),
+          icon: "copy.svg",
+        },
+      ]
```

Comment:
Why another variable? Just add this duplicate waala object above where options are defined?